web/widgets.py

```
@@ -135,3 +139,224 @@ def __init__(self, attrs=None):
+                    // Initial render
+                    renderItems();
+                }});
+            </script>
+        </div>
+        """.format(
+            name,
+            "".join(feature_items),
+            json.dumps(decoded_value),  # {0}  # {1}  # {2}
+        )
+
+    def value_from_datadict(self, data: dict, files: dict, name: str) -> list:
```

Ruff (0.8.2)

Unused method argument: `files`

(ARG002)